Kiosk/App/SwiftExtensions.swift

```diff
@@ -54,3 +54,19 @@ extension Optional where Wrapped: Occupiable {
         return !isNilOrEmpty
     }
 }
+
+// TODO: PR this into Moya
```

Comment:
I have so PR'd: https://github.com/Moya/Moya/pull/401

However, we're a major version of Moya behind, and I don't have time to update right now. We'll put this on the back-burner for now.